learner_utils.py

```diff
@@ -28,6 +28,22 @@ def learner_decisions(subpops, current = None, min_fn=quadratic_min):
             new_thetas.append(theta_i)
     return new_thetas
 
+def learner_max_price(thetas, prices, subpops, min_fn=quadratic_min):
+    n_learners = len(thetas)
+    new_prices = [0] * len(prices)
+    #price_candidates = np.linspace(0, 100, 51)
+    for i in range(n_learners):
+        price_candidates = [prices[i] + 1, prices[i] * 2, prices[i] / 2]
```

Comment:
will add the identity price `prices[i]` as an option (I didn't realize I had forgotten to do that before)